azurelinuxagent/common/logcollector.py

```diff
@@ -51,7 +52,7 @@
     os.path.join(_AGENT_LIB_DIR, "history", "*.zip"),
     os.path.join(_EXTENSION_LOG_DIR, "*", "*"),
     os.path.join(_EXTENSION_LOG_DIR, "*", "*", "*"),
-    "/var/log/waagent*",
+    "{0}*".format(_AGENT_LOG)
```

Comment:
this will also capture waagent.log.. you may want to use 'waagent\..+' instead

Reply:
Thanks, it's more specific with your suggestion. Updated.